packages/mobile/src/in-house-liquidity/index.test.ts

```diff
@@ -0,0 +1,149 @@
+import { createPersonaAccount, getAuthAndDateHeaders, signAndFetch } from 'src/in-house-liquidity'
+import { FetchMock } from 'jest-fetch-mock/types'
+import { mockMnemonic } from 'test/values'
+import * as signatureUtils from '@celo/utils/lib/signatureUtils'
+import networkConfig from 'src/geth/networkConfig'
+
+jest.mock('@celo/utils/lib/account', () => {
+  return {
+    generateKeys: jest.fn(() => ({
+      address: MOCK_USER.walletAddress,
+      privateKey: MOCK_USER.privateKey,
+      publicKey: MOCK_USER.publicKey,
+    })),
+  }
+})
+
+jest.mock('src/backup/utils', () => ({
+  ...(jest.requireActual('src/backup/utils') as any),
+  getStoredMnemonic: jest.fn(() => mockMnemonic),
+}))
+
+const signMessage = jest.spyOn(signatureUtils, 'signMessage')
+
+const MOCK_USER = {
+  walletAddress: '0x2Ec2c5D904ed2964F791aDA2185B9c2241C371c0',
+  accountMTWAddress: '0x8b2f81f995c910aede52116406452468946d7d52',
+  privateKey: '7241c1f13452c1990acc737228dd8d4873ccaf14287c8ee0a59cdb3e07d7526d',
+  publicKey: '02a54b261ef5f8c8f97b8fe8d867d00dab35b5f7b39441691b05382b6d47bf709b',
+}
+
+describe('In House Liquidity Calls', () => {
+  const mockFetch = fetch as FetchMock
+
+  const date = new Date('2021-05-14T11:01:58.135Z')
+  const expectedDateString = date.toUTCString()
+
+  beforeEach(() => {
+    mockFetch.resetMocks()
+    jest.spyOn(global, 'Date').mockImplementationOnce(() => date as any)
+    jest.clearAllMocks()
+  })
+
+  describe('getAuthAndDateHeaders', () => {
+    it('creates the correct headers for a GET request', async () => {
+      const headers = await getAuthAndDateHeaders(
+        'GET',
+        '/account/foo',
+        MOCK_USER.accountMTWAddress
+      )
+
+      // Creates the correct Headers
+      expect(headers).toMatchObject({
+        Date: expectedDateString,
+        Authorization: `Valora ${MOCK_USER.walletAddress}:0x1c8689cf5de841119091ef4f0c49170c4130909b402dfba1af7bcab4f047a5f23735d0fe8ab631804cd51a098bf335cd2a8243d94630435e6d854fc96998f8a313`,
```

Comment:
nit: extract the expected serialized signature into its own variable for readability